src/app.ts

```diff
@@ -27,6 +27,15 @@ const app = express();
 
 log.info("Started.");
 
+const appPort = config.server.port || 3000;
+
+if (!config.server.port) log.warn("No port specified. Using default: 3000");
+
+if (!/^([0-9]{1,4}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$/.test(appPort)){
```

Comment:
Why this complicated regex (I'm not even in the mood to check the possible valid inputs) instead of just checking that it's between x and 65535?

Reply:
because I was bored. It works just fine tho...

https://regexr.com/3suel

Reply:
Actually.. Nope xD

regexr.com/3suf4

Reply:
Or.. You could just stick to a simple range check xD that regex makes it unnecessarily complicated to see that it actually will accept anything from 1 to 65535 or if it filters stuff like 80 or something